autopilot/contractor.go

```diff
@@ -551,11 +552,6 @@ func (c *contractor) runContractRefreshes(ctx context.Context, cfg api.Autopilot
 	for _, ci := range toRefresh {
 		// TODO: keep track of consecutive failures and break at some point
 
-		// break if the contractor was stopped
-		if c.isStopped() {
```

Comment:
same question

Reply:
resolving, duplicate of https://github.com/SiaFoundation/renterd/pull/129/files#r1098891311 tackling it there